core/domain/feedback_services.py

```diff
@@ -280,9 +427,17 @@ def get_closed_threads(exploration_id, has_suggestion):
 
 
 def get_all_threads(exploration_id, has_suggestion):
-    """Returns a list of all threads with suggestions if has_suggestion is True;
-    otherwise, returns a list of all threads with no suggestions."""
+    """Fetch all threads of the given exploration id
```

Comment:
Fetches all threads (regardless of their status) that correspond to the given exploration id.

(Idea: trying to make clear that, unlike the previous functions, status doesn't matter here. Also, using "Fetches" for consistency with the others and the style guide; added full stop at end.)